samples/samples/backup_sample.py

```diff
@@ -34,7 +34,8 @@ def create_backup(instance_id, database_id, backup_id):
 
     # Create a backup
     expire_time = datetime.utcnow() + timedelta(days=14)
-    backup = instance.backup(backup_id, database=database, expire_time=expire_time)
+    version_time = database.earliest_version_time()
```

Comment:
`earliest_version_time` is a property not a method.
```suggestion
    version_time = database.earliest_version_time
```